afs/media/js/views/components/workflows/analysis-areas-annotation-step.js

```diff
@@ -174,11 +176,15 @@ define([
             self.selectAnalysisAreaInstance(newTile);
         };
 
-        // this.saveTile = function() {
-        //     self.tile.save().then(function(data) {
-        //         self.observationInstances(self.card.tiles())
-        //     });
-        // };
+        this.saveAnalysisArea = function () {
```

Comment:
nit: can this be named something like `saveWorkflowStep`?

Reply:
Makes much more sense after today